LICENSE.md

```diff
@@ -18,6 +18,13 @@ relative subfolders:
 * [mve_intrinsics](mve_intrinsics)
 * [neon_intrinsics](neon_intrinsics)
 
+# License for the legacy releases
+
+All the legacy ACLE releases are licensed under an Arm proprietary license.
+They predate the new license of the current versions of the documents.
```

Comment:
as opposed to the `current` here which frames the current versions against the proprietary licensed documents

Reply:
Fixed as already mentioned.